tool/templates/kotlin/Struct.kt.jinja

```diff
@@ -27,6 +28,55 @@ internal class {{type_name}}Native: Structure(), Structure.ByValue {
 
 {% endif -%}
 
+{% if !callback_params.is_empty() -%}
```

Comment:
issue: callbacks can exist on structs, enums, or opaques, we should support all three.

you may wish to use an included template to make it easy

Reply:
Oh, good point. I didn't think of that. I'll add tracking for those other custom types too.